src/test-utils.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { Provider } from 'react-redux';
+
+import { initializeStore } from 'store';
+
+export const executeThunk = async (thunk, dispatch, getState) => {
+  await thunk(dispatch, getState);
+};
+
+export function renderWithProviders(
+  ui,
+  {
+    preloadedState = {},
+    // Automatically create a store instance if no store was passed in
+    store = initializeStore(preloadedState),
+    ...renderOptions
```

Comment:
```suggestion
    ...renderOptions,
```
Trailing comma.

Reply:
This change makes failed redux test: Unexpected trailing comma after rest element. So I won't add it